src/app/dashboard/admins/verification/page.js

```
@@ -0,0 +1,226 @@
+
+    const formatDate = (timestamp) => {
+        if (!timestamp) return 'Date inconnue';
+
+        // Convert Firestore timestamp to JS Date
+        const date = new Date(timestamp._seconds * 1000);
+        return date.toLocaleDateString('fr-FR');
+    };
+
+    // Filter users based on the search query
+    const filteredUsers = users.filter(user =>
+        (user.name && user.name.toLowerCase().includes(searchQuery.toLowerCase())) ||
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)